src/proc/colorizer.cpp

```diff
@@ -150,19 +150,41 @@ namespace librealsense
 
         _maps = { &jet, &classic, &grayscale, &inv_grayscale, &biomes, &cold, &warm, &quantized, &pattern, &hue };
 
-        auto min_opt = std::make_shared<ptr_option<float>>(0.f, 16.f, 0.1f, 0.f, &_min, "Min range in meters");
+        auto min_opt = std::make_shared< ptr_option< float > >(0.f, 16.f, 0.1f, 0.f, &_min, "Min range in meters");
 
-        auto max_opt = std::make_shared<ptr_option<float>>(0.f, 16.f, 0.1f, 6.f, &_max, "Max range in meters");
+        auto max_opt = std::make_shared< ptr_option< float > >(0.f, 16.f, 0.1f, 6.f, &_max, "Max range in meters");
 
-        register_option(RS2_OPTION_MAX_DISTANCE,
-            std::make_shared<max_distance_option>(
-                max_opt,
-                min_opt));
+        auto max_dist_opt = std::make_shared< max_distance_option >(max_opt, min_opt);
 
-        register_option(RS2_OPTION_MIN_DISTANCE,
-            std::make_shared<min_distance_option>(
-                min_opt,
-                max_opt));
+        auto min_dist_opt = std::make_shared< min_distance_option >(min_opt, max_opt);
+
+        register_option(RS2_OPTION_MAX_DISTANCE, max_dist_opt);
+
+        register_option(RS2_OPTION_MIN_DISTANCE, min_dist_opt);
+
+        auto hist_opt = std::make_shared< ptr_option< bool > >(false,
+            true,
+            true,
+            true,
+            &_equalize,
+            "Perform histogram equalization");
+        register_option(RS2_OPTION_HISTOGRAM_EQUALIZATION_ENABLED, hist_opt);
```

Comment:
You've changed the order... any specific reason? Does that also change the order in the GUI?

Reply:
Changed the order because I need to have `hist_opt` to create the observer functions. It doesn't affect the GUI

Reply:
The register is the order I mean -- you can create the option, pass it along, but register it in the same order as before, no?

Reply:
Probably, if you prefer it that way